workflow/scripts/plot_annotatepeaks_summary_homer.R

```diff
@@ -0,0 +1,18 @@
+log <- file(snakemake@log[[1]], open="wt")
+sink(log)
+sink(log, type="message")
+
+library("tidyverse")
+
+homer_data <- read_tsv(snakemake@input[[1]])
+homer_data <- homer_data %>% gather(`exon`, `Intergenic`, `intron`, `promoter-TSS`, `TTS`, key="sequence_element", value="counts")
+
+peaks_sum <- ggplot(homer_data, aes(x = counts, y = sample, fill = sequence_element)) +
+  geom_bar(position="fill", stat="Identity") +
+  theme_minimal() +
+  labs(x="", y="Peak count") +
+  theme(legend.position = "none") +
```

Comment:
If I understand this correctly, this means that no legend / `guide` is provided to indicate which `fill` color refers to which sequence element, right? If this was to mimic the original image in the nextflow workflow, I would deviate here to have a useful guide included directly in the figure. This makes it easier to reuse a PDF figure for publication directly. Or am I missing some other important point, here?